extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -460,10 +464,14 @@ export class AmpA4A extends AMP.BaseElement {
         /** @return {?Promise<?{bytes: !ArrayBuffer, headers: !Headers}>} */
         .then(fetchResponse => {
           checkStillCurrent(promiseId);
-          if (!fetchResponse || !fetchResponse.arrayBuffer) {
-            return null;
-          }
           this.protectedEmitLifecycleEvent_('adRequestEnd');
+          // If the response has response code 204, or is null, collapse it.
+          if (!fetchResponse
+              || !fetchResponse.arrayBuffer
+              || fetchResponse.status == 204) {
+            this.forceCollapse();
+            return Promise.reject(NO_CONTENT_RESPONSE);
```

Comment:
@tdrl should we send a CSI beacon here to help how often we do not execute any render flow due to this?